MonacoEditorComponent/Monaco/Editor/AcceptSuggestionOnEnter.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Newtonsoft.Json;
```

Comment:
Are these all the new auto-generated files?

Should we label them as such or exclude them from the repo?

Reply:
They're auto-generated but with some modifications because the generator will generate double type for all number types in TypeScript. I don't think they should be excluded from the repo, however, we can update those models periodically using type bindings generator.

Reply:
@hez2010 alright, it could be good to add that process to the readme or another doc somewhere so that when we want to take the next update to Monaco anyone can know how to re-run, compare, and integrate changes.